synapse/storage/devices.py

```diff
@@ -458,6 +465,21 @@ def get_user_whose_devices_changed(self, from_key):
         rows = yield self._execute("get_user_whose_devices_changed", None, sql, from_key)
         defer.returnValue(set(row["user_id"] for row in rows))
 
+    def get_users_and_hosts_device_list_changes(self, from_key):
```

Comment:
I wonder if the name of this method should indicate that this is for outbound pokes?